external/src/main/java/com/redhat/cloud/policies/engine/LightweightEngineImpl.java

```diff
@@ -138,7 +144,11 @@ public void process(Event event) {
             // In the old implementation, the time comes from the Action constructor which relies on System.currentTimeMillis().
             LocalDateTime nowUTC = LocalDateTime.ofInstant(Instant.ofEpochMilli(System.currentTimeMillis()), UTC);
 
-            PoliciesAction policiesAction = new PoliciesAction();
+            PoliciesAction policiesAction = new PoliciesAction(useOrgId);
+
+            // TODO POL-650 is tenantId also the orgID? I guess it will become the ordId.
+            policiesAction.setOrgId(event.getTenantId());
```

Comment:
Maybe depending on the `useOrgId` value we would set it on the OrgId or the AccountId

Reply:
thought about this as well. I did it this way to keep policies-engine compatible in every case. But maybe that's not needed. Not sure. Maybe @gwenneg has got an opinion about it?